Add missing #include directives to support building in strict(er)
C99 mode.

diff --git a/execunix.c b/execunix.c
index 6cde87a1b008c54a..83bb8978878be294 100644
--- a/execunix.c
+++ b/execunix.c
@@ -44,6 +44,8 @@
 # include <errno.h>
 
 # ifdef USE_EXECUNIX
+# include <unistd.h>
+# include <sys/wait.h>
 
 # ifdef OS_OS2
 # define USE_EXECNT
diff --git a/fileunix.c b/fileunix.c
index 41127a293aca58ea..54014b28c92da4d0 100644
--- a/fileunix.c
+++ b/fileunix.c
@@ -38,6 +38,7 @@
 # include "pathsys.h"
 
 # ifdef USE_FILEUNIX
+# include <unistd.h>
 
 # if defined( OS_SEQUENT ) || \
      defined( OS_DGUX ) || \
@@ -51,7 +52,6 @@
      defined( OS_NEXT )
 /* need unistd for rhapsody's proper lseek */
 # include <sys/dir.h>
-# include <unistd.h>
 # define STRUCT_DIRENT struct direct 
 # else
 # include <dirent.h>
diff --git a/make1.c b/make1.c
index b12ca3cef924da3b..c51dabfd58fb7b63 100644
--- a/make1.c
+++ b/make1.c
@@ -62,6 +62,8 @@
 # include "command.h"
 # include "execcmd.h"
 
+# include <unistd.h>
+
 static void make1a( TARGET *t, TARGET *parent );
 static void make1b( TARGET *t );
 static void make1c( TARGET *t );
